tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -99,3 +99,39 @@ describe('Test GitRepo Bundle name validation and max character trimming behavio
     }
   )
 });
+
+describe('Test resource behavior after deleting GitRepo using keepResources option', { tags: '@p1'}, () => {
+  const repoName = "local-cluster-fleet-71"
+  const branch = "master"
+  const path = "qa-test-apps/nginx-app"
+  const repoUrl = "https://github.com/rancher/fleet-test-data/"
+  const appNamespace = 'nginx-keep'
+  const appName = 'nginx-keep'
+  const keepResourceData: testData[] = [
+    { qase_id: 69,
+      keepResources: 'true',
+      test_explanation: 'RESOURCES will NOT be DELETED after GitRepo is deleted.',
+    },
+    { qase_id: 70,
+      keepResources: 'false',
+      test_explanation: 'RESOURCES will be DELETED after GitRepo is deleted.',
+    },
+  ]
+  keepResourceData.forEach(
+    ({ qase_id, keepResources, test_explanation}) => {
+      qase(qase_id,
+        it(`Test ${test_explanation}`, { tags: `@fleet-${qase_id}` }, () => {
+          cy.fleetNamespaceToggle('fleet-local')
+          cy.addFleetGitRepo({ repoName, repoUrl, branch, path, keepResources });
+          cy.clickButton('Create');
+          cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+          cy.deleteAllFleetRepos();
+          if (keepResources === 'true') {
```

Comment:
It is very nice the check if `true`, but it would be nice to also test the `false` case on a separate pr to insure it is really not kept.

Reply:
I will try update in the same PR else, I will create new one.